recipes-support/libunwind/libunwind_1.1.bbappend

```diff
@@ -0,0 +1,9 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+
+# Add necessary patches according to:
+# https://github.com/dotnet/coreclr/blob/master/Documentation/building/linux-instructions.md#build-for-armlinux
+
+SRC_URI_append += "\
+           file://libunwind.git-770152268807e460184b4152e23aba9c86601090.patch \
+           file://expand-arm-Validate-memory-before-access.patch \
+"
```

Comment:
My setup doesn't have `libunwind_1.1.bb` anywhere. The only recipe I have is the one in `poky/meta/recipes-support/libunwind/libunwind_git.bb`. I checked the poky history and the `krogoth` branch is the last one having the `libunwind_1.1.bb` recipe.